lib/crypto.js

```diff
@@ -515,7 +515,7 @@ DiffieHellman.prototype.setPrivateKey = function(key, encoding) {
 
 function ECDH(curve) {
   if (typeof curve !== 'string')
-    throw new TypeError('curve should be a string');
+    throw new TypeError('"curve" argument should be a string');
```

Comment:
Small question: Every other error is something line ".... must be ...". Does it make sense to change it also in crypto?